.markdownlint.json

```diff
@@ -0,0 +1,5 @@
+{
+    "MD013": false,
```

Comment:
rant: I wish JSON supported comments, 
so that we could describe what these rules actually refer to.
(not at all blocking, just a rant)